docs/source/howto/data.rst

```diff
@@ -52,6 +52,103 @@ Deleting data
 
 `#3999`_
 
+Serving your data to others
+===========================
```

Comment:
@ramirezfranciscof please update this section to respect the new style rules: i.e., one sentence per line and no arbitrary line breaks